src/main/java/uk/gov/justice/hmpps/casenotes/dto/CaseNoteFilter.kt

```diff
@@ -27,4 +27,7 @@ class CaseNoteFilter(
 
   @Schema(description = "Filter by username", example = "USER1")
   val authorUsername: String? = null,
+
+  @Schema(description = "Filter by a list of case note types and optional case not sub types separated by plus", example = "KA-KE,OBS,POM-GEN")
```

Comment:
```suggestion
  @Schema(description = "Filter by a comma separated list of case note types and optional case note sub types separated by minus", example = "KA-KE,OBS,POM-GEN")
```

Reply:
this is also interesting - why not get spring to do it for you and pass an array of strings in?  Should work as http parameters can be duplicated.  Seems odd to have to comma separate here instead